group_vars/matrix_servers

```diff
@@ -2370,6 +2376,51 @@ matrix_hookshot_public_hostname: "{{ matrix_server_fqn_matrix }}"
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-bridge-zulip
+#
+######################################################################
+
+# We don't enable bridges by default.
+matrix_bridge_zulip_enabled: false
```

Comment:
Please set owner to `{{ matrix_admin }}` here as well - to keep in sync with other bridges

Reply:
Will it work though Heisenbridge does not have it?

Reply:
Addressed with https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/4695/commits/84ad18b4baae0c64cbc36296f17f389d27ed4658.